_includes/manuals/1.16/3.5.2_configuration_options.md

```diff
@@ -3,6 +3,8 @@ Configuration is broken into two parts. The */etc/foreman/settings.yaml* file an
 
 The configuration file can also override those settings specified in the web interface.  Any settings added in the config file that are available in the web interface will be made read-only.
 
+If the default installation was made, using `foreman-installer`, then the configuration bellow, will already be set.
```

Comment:
I'm not sure the commas should be there, but would be good to have this confirmed by some native speaker, @ehelms perhaps?

Reply:
The comma's are indeed not needed. There is a typo in the word `below` as well (an extra 'l').